contract/src/lib.rs

```diff
@@ -277,14 +317,16 @@ impl MpcContract {
         payload: [u8; 32],
         depth: usize,
     ) -> PromiseOrValue<(String, String)> {
+        log!("sign_helper: payload={:?} depth={}", payload, depth);
```

Comment:
don't think this log is necessary since we already have one later with "signature not ready yet" along with the depth. The panic won't give us any logs anyways